tests/capabilities/test-class-capability-manager.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+class WPSEO_Capability_Manager_Test extends WPSEO_Abstract_Capability_Manager {
+	/**
+	 * Adds the registerd capabilities to the system.
+	 */
+	public function add() {
+
+	}
+
+	/**
+	 * Removes the registered capabilities from the system
+	 */
+	public function remove() {
+
```

Comment:
Empty test.

Reply:
This is not a test, this is an implementation of the interface.